src/crimson/os/seastore/async_cleaner.h

```diff
@@ -642,10 +652,24 @@ class JournalTrimmerImpl : public JournalTrimmer {
   trim_ertr::future<> trim_alloc();
 
   journal_seq_t get_tail_limit() const;
+  journal_seq_t get_dirty_tail_min_target() const;
   journal_seq_t get_dirty_tail_target() const;
+  journal_seq_t get_trim_dirty_target() const;
```

Comment:
Probably easier to distinguish `get_dirty_tail_target()` if rename `get_trim_dirty_target()` to `get_dirty_tail_target_per_cycle()` ?